index.html

```diff
@@ -0,0 +1,328 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous">
+</head>
+
+<body>
+<!-- START Header -->
+<header>
+    <nav id="top-navbar" class="flex">
+        <a href="#" id="leftTopNav">
+            <img src="./Images/tnw.svg" alt="">
+        </a>
+        <div id="rightTopNav" class="flex">
+            <a href="#" class="topNav-icons flex" id="right-top-nav-a">
+                <i class="fas fa-search"></i>
+                <i class="fas fa-bars"></i>
+            </a>
+        </div>
+        <div id="container-small-hiden-nav" class="small-screen-hidden">
+            <div id="top-nav-hidden" class="flex">
+                <ul class="nav-ntw flex">
+                    <li><a id="hiden-nav-a" href="#">News</a></li>
+                    <li><a href="#">Business▾</a></li>
+                    <li><a href="#">AMAs</a></li>
+                    <li><a href="#">Spaces</a></li>
+                    <li><a href="#">Terms & Conditions</a></li>
+                </ul>
+                <ul class="flex">
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Advertise</a></li>
+                    <li><a href="#">Jobs</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+            <div id="second-nav-hidden" class="small-screen-hidden flex">
+                    <ul id="list-second-nav">
+                        <li><a href="#">LATEST</a></li>
+                        <li><a href="#">HARD FORK</a></li>
+                        <li><a href="#">PLUGGED</a></li>
+                        <li><a href="#">README</a></li>
+                        <li><a href="#">GROWTH QUARTERS</a></li>
+                    </ul>
+                <div>
+                    <a href="#" id="second-hidden-a" class="topNav-icons flex">
+                        <i class="fas fa-search"></i>
+                    </a>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <a href="#" id="topData">
+        <span>Calling young digital talent in the Netherlands – Be recognized in the T500 →</span>
+    </a>
+</header>
+<!-- END Header -->
+
+<!-- Start Highlights -->
+<section class="first-section">
+    <div class="grid-container grid">
+        <div class="first-grid highlights">
```

Comment:
Better to use article tag here instead of div. Apply same changes for the other highlights article elements

Reply:
Thank you for the review and suggestions. We corrected the parts you mentioned.

Reply:
We checked the font awesome icons, and they are functioning normally.